sdk/storage/azure-storage-file-share/pyproject.toml

```diff
@@ -3,3 +3,4 @@ mypy = false
 pyright = false
 type_check_samples = false
 verifytypes = false
+strict_sphinx = true
```

Comment:
Did you want to update the `pyproject.toml` for changefeed as well?

Reply:
This is a good catch, it should have.